src/www/page/load.tsx

```diff
@@ -10,6 +10,78 @@ const LoadPage: React.FunctionComponent = () => {
     const [measurementsPath, setMeasurementsPath] = useState('');
     const [normalize, setNormalize] = useState<boolean>(false);
     const [files, setFiles] = useState<string[]>([]);
+    const [baseMeasurement, setBaseMeasurement] = useState('');
+
+    // Get all file paths
+    const getFilePaths = (filesArray: Array<string>) => {
+        let paths: Array<string> = []
+        filesArray.forEach(file => {
+            let fullPathMeasurementFile = measurementsPath + '\\' + file;
```

Comment:
Can be made cross-platform by using [path.join](https://nodejs.org/api/path.html#path_path_join_paths)